apps/content/tests/tenant/test_reciter_list.py

```
@@ -104,17 +81,18 @@ def test_list_reciters_should_return_only_active_reciters_with_ready_recitations
 
         reciter_item = items[0]
         self.assertEqual(self.active_reciter.id, reciter_item["id"])
-        # Only the valid RECITATION asset with READY + RECITATION resource should be counted
+        # Only the valid RECITATION asset with READY status should be counted
         self.assertEqual(1, reciter_item["recitations_count"])
 
     def test_list_reciters_ordering_by_name(self):
         # Arrange – another active reciter so we can test ordering
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)